packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx

```diff
@@ -40,6 +42,14 @@ const DigitalContactsCodeVerificationContext = createContext<
 >(undefined);
 
 const DigitalContactsCodeVerificationProvider: FC<ReactNode> = ({ children }) => {
+  const { t } = useTranslation(['common', 'recapiti']);
+  const digitalAddresses = useAppSelector(
```

Comment:
Vedere se eventualmente passare i digitalAddresses dall'esterno (in questo caso dalla Contacts.page)